src/constants/fieldMaps.ts

```diff
@@ -172,6 +174,24 @@ export const awsRegions = [
   },
 ];
 
+const listOfDateFormatStrings = [
+  "MM-dd-yyyy",
+  "dd-MM-yyyy",
+  "yyyy-MM-dd",
+  "MM/dd/yyyy",
+  "dd/MM/yyyy",
+  "yyyy/MM/dd",
+  "MMM d, yyyy",
+];
+
+export const dateFormats = [...listOfDateFormatStrings].map((format) => ({
```

Comment:
hoisted by my own petard
```suggestion
export const dateFormats = listOfDateFormatStrings.map((format) => ({
```